src/lib/timing.c

```diff
@@ -152,10 +176,13 @@ void xdebug_nanotime_init(void)
 	context.last_abs = 0;
 #if PHP_WIN32 | __APPLE__ | defined(CLOCK_MONOTONIC)
 	context.start_rel = xdebug_get_nanotime_rel(&context);
-	context.last_rel = 0;
 #endif
 
 	XG_BASE(nanotime_context) = context;
+
+#if WIN_SUPPORTS_RDTSC
+    xdebug_nanotime_init_calibrate_rdtsc();
```

Comment:
Even if we support RDTSC, we should wrap this in if and enable it based on enviroment variable/config. Please advise.